.github/workflows/build.yml

```diff
@@ -10,6 +10,7 @@ on:
   pull_request:
     branches: ['main', 'release/**']
     paths:
+      - '.github/workflows/build.yml'
```

Comment:
we should include `prebuild.yml` as well.

Reply:
Wouldn't doing this also make it so changes in `prebuild.yml` would make `build.yml` also run?

Reply:
Oh, right. I guess I was thinking prebuild should rerun if prebuild changes, but that's not what this is. Whoops.